aiidalab/git_util.py

```diff
@@ -92,12 +92,12 @@ def get_branch_tracking_status(self, branch: bytes) -> BranchTrackingStatus | No
                 return BranchTrackingStatus.EQUAL
 
             # Check if local branch is behind the tracked branch:
-            for commit in self.get_walker(self.refs[tracked_branch]):
+            for commit in self.get_walker([self.refs[tracked_branch]]):
```

Comment:
These calls were working because of this code path in dulwich:

https://github.com/jelmer/dulwich/blob/3376d1f99806323305ee4666146367bdcda7d02e/dulwich/walk.py#L281